### PR TITLE
Create galatasaray-universitesi-sosyal-bilimler-enstitusu.csl

### PR DESCRIPTION
**Pull request by mtarakci**
This is for the Institute of Social Sciences of Galatasaray University. 

The thesis manual can be reached in Turkish at: 
http://dosya.gsu.edu.tr/Sayfalar/2019/2/tez-yazim-kilavuzu-645.doc

### CODE REVIEW

**Comment by mtarakci** [comment]
I couldn't figure out what the problem is.

File: galatasaray-universitesi-sosyal-bilimler-enstitusu.csl [hunk 1]
```diff
@@ -0,0 +1,1440 @@
...
+    </author>
+    <category citation-format="note"/>
+    <category field="social_science"/>
+    <summary xml:lang="tr">Galatasaray Üniversitesi Sosyal Bilimler Enstitüsü'nde yüksek lisans ve doktora tezi yazan öğrencilerimiz için tez yazım kılavuzuna uygun olarak hazırlanmıştır.</summary>
+    <updated>2019-02-20T16:52:22+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="tr">
+    <terms>
+      <term name="editor" form="verb-short">ed.</term>
```
Comment (adam3smith) [inline galatasaray-universitesi-sosyal-bilimler-enstitusu.csl]: A lot of these terms already exist in the locale file (where you provided some of the translations) I'd rather not duplicate them here. Could you please remove all the ones that are identical in the locale?
(Or is this a Mendeley problem again -- in which case this is really up to them to fix. The locale was updated almost a year ago)

Comment (mtarakci) [reply galatasaray-universitesi-sosyal-bilimler-enstitusu.csl]: Dear Adam Smith, 
I have removed some of the local terms. Thank you for your attention.

**Comment by mtarakci** [comment]
I have already finished, and am waiting it to be merged.

**Comment by adam3smith** [comment]
Thanks!

**Pull request merged**